### ISSUE TITLE
Could not pass event PlayerJoinEvent to LuckPerms v5.3.20

### ISSUE THREAD

**Issue by Dreeam-qwq** [lucko/LuckPerms#2970]
### Description

```
Could not pass event PlayerJoinEvent to LuckPerms v5.3.20
java.lang.IncompatibleClassChangeError: Found interface us.myles.ViaVersion.ViaManager, but class was expected
```

### Reproduction steps

1. when player login in server
2. See error

### Expected behaviour

https://pastebin.com/kuBxDpWf

### Environment details

* Server type/version: `Paper` running version `1.12.2` build `1618`
* LuckPerms version: `v5.3.20`
* Proxy: `Waterfall`

### Any other relevant details

`Viaversion, Viabackwards, Viarewind, Protocllib` are latest dev builds.

**Comment by underscore11code** [comment]
> `Viaversion, Viabackwards, Viarewind, Protocllib` are latest dev builds.

Is this reproducible on latest stable for these? This looks like an issue on their end, not LP.

**Comment by benwoo1110** [comment]
I think I have seen this before, it's probably due to their latest dev build having API breakages since the release version works from what I know.

**Comment by lucko** [comment]
Should be fixed in v5.3.24 and newer :)
